scripts/Package_BGA/bga.py

```diff
@@ -166,8 +157,9 @@ def crtYdRound(x):
     parser.add_parameter("pkg_height", type=float, required=True)
     parser.add_parameter("pitch", type=float, required=True)
     parser.add_parameter("pad_diameter", type=float, required=True)
+    parser.add_parameter("mask_margin", type=float, required=False, default=0)
+    parser.add_parameter("paste_margin", type=float, required=False, default=0)
     parser.add_parameter("paste_ratio", type=float, required=False, default=0)
-    parser.add_parameter("paste_diameter", type=float, required=False, default=0)
```

Comment:
Since you changed this option, be sure to update the .yml files to match.  It's only used in csp.yml I believe.